README.md

```diff
@@ -222,6 +222,25 @@ visible in the search UI. :clap:
 >:bulb: Just like with Redis: if you share the same Elasticsearch, the messages from everyone
 > will be visible and searcheable to every server and its users.
 
+### Side-Quest: Who am I? (SIDE-QUEST-5) 
+
+Now we want to practice what we have done so far and add another info into our [SearchableChatMessage](src/main/java/search/SearchableChatMessage.java).
+First we have to look at the [client.js](src/main/resources/assets/client/client.js) and read the info (e.g. the user agent string) from the user's browser
+and add it to every message we send through the web socket. Then we have a new field to [SearchableChatMessage](src/main/java/search/SearchableChatMessage.java)
```

Comment:
Suggestion:
First we want to enhance the client.js to retrieve an additional information from the user's browser (e.g. the user agent header) and send it over the web socket. Now we have to include a new field in SearcheableChatMessage to store the new information (see how existing fields are declared) and ultimately display it by by editing the search.html.pasta...

Mention the /system/tags to list the possible Tagliatelle tags, maybe add a small hint to which tags are to be used to accomplish the task?